docs/content/plugins/notifications/sendgrid.mdx

```diff
@@ -3981,6 +3981,24 @@ You can also track analytics related to emails sent from the SendGrid dashboard.
 
 ---
 
+## Dynamic usage
+
+You can resolve the SendGrid service to send emails from your custom services or other resources. For example:
+
+```
```

Comment:
Sorry forgot to mention this before, but can you add the language here `ts`? You might also need to run the command `yarn lint:docs --fix` to resolve any eslint errors

Reply:
Oops. Missed that. Thanks.